test/e2e/core/kbdFocusLayerMenu.test.js

```diff
@@ -0,0 +1,75 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe('Focus stays on checkboxes when using keyboard navigation', () => {
```

Comment:
test looks good, and works well! since its a small bug we should probably integrate it into an existing test rather than a new test file. WDYT about putting it at [layerControl.test.js](https://github.com/Maps4HTML/MapML.js/blob/main/test/e2e/layers/layerControl.test.js)?

Reply:
That should work! We should be able to run these two particular tests in slowmo without slowing down the rest of that test file :)